twoliter/src/tool-crates/embedded-bundle/build.rs

```diff
@@ -14,7 +14,7 @@ use std::path::{Path, PathBuf};
 use std::time::SystemTime;
 use std::{env, fs};
 
-const DATA_INPUT_DIR: &str = concat!(env!("CARGO_MANIFEST_DIR"), "/embedded");
+const DATA_INPUT_DIR: &str = concat!(env!("CARGO_MANIFEST_DIR"), "/../../../embedded");
```

Comment:
nit: I don't *love* having this path point outside of the crate, how would you feel about moving the embedded files into this crate?

Reply:
I'm OK with doing that, or even symlinking it. I just know that I end up opening `twoliter/embedded` very frequently, so I'm reluctant to bury it.

Reply:
yeah I agree with that, it's a path I'm pretty used to as well. I'd be in favor of moving the directory into this crate and symlinking to it at `twoliter/embedded`

Reply:
I agree that that's the correct ownership of the data, but I may make the symlink point the other way so that the GitHub UI also continues to show `twoliter/embedded`